README.md

```diff
@@ -66,47 +66,48 @@ Similarly to the hostname based setup, append the commands to `.bashrc` to set t
 *You only need to run these commands if this is the first time you you run this demo with the current user PC or ROBOTONT on-board computer.*
 
 **On Robotont on-board computer**, install ROS wrapper of RTAB-Map<br/>
-```sudo apt update```
+```sudo apt update```<br/>
 ```sudo apt install ros-melodic-rtabmap-ros```
```

Comment:
I suggest using the same style as shown above:
```bash
code line 1
code line 2
```
In that case, no <br> magic has to be included.
I generally agree that <br> is the best way to break lines if neccessary.
Also, it does not hurt to specify it is a bash command :-)